changes/3559.added

```diff
@@ -1,2 +1,2 @@
-Added `HardwareFamily` model class.
-Added `hardware_family` field to Device Type model class.
+Added `DeviceFamily` model class.
```

Comment:
I think these are changes for issue #3559, and should be left that way?

Reply:
We don't have a defined standard for maintaining "pre-release" changelog fragments, but I understand the reasoning here - it's potentially confusing to the user when we ship 2.2 for the "added" to refer to things that weren't added as described. On the flip side, it's good for the change fragments to document the actual work done. Perhaps something like the following might be a good compromise?

```suggestion
Added `HardwareFamily` model class. (Renamed before release to `DeviceFamily`.)
```